packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2870,6 +2893,20 @@ class SemanticsNode with DiagnosticableTreeMixin {
   SemanticsRole get role => _role;
   SemanticsRole _role = _kEmptyConfig.role;
 
+  /// {@template flutter.semantics.SemanticsNode.controlsVisibilityOfNodes}
+  /// A set of [SemanticsNode.identifier]s' visibilities that this node
+  /// controls
+  ///
+  /// If a widget is controlling the visibility of the other widget, for example,
+  /// [Tab]s controls children visibilities of [TabBarView] or [ExpansionTile]
+  /// controls visibility of its expanded content. One must assign a
+  /// [SemanticsNode.identifier] to the hidden content and also provide a
+  /// set of identifiers include the hidden content's identifier to this
+  /// property.
```

Comment:
Same nits as for `SemanticsProperties.controlsVisibilityOfNodes`. But also, should we apply this template in `SemanticsProperties.controlsVisibilityOfNodes` and avoid duplicating the text?